src/tunacode/tools/write_file.py

```
@@ -54,7 +52,7 @@ async def _run_write_file(filepath: str, content: str) -> str:
         file_obj.write(content)
 
     result = f"Successfully wrote to new file: {filepath}"
-    return await maybe_prepend_lsp_diagnostics(result, Path(filepath))
+    return result
```

Ruff (0.15.10)

[warning] Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)